concordia/static/scss/action-app.scss

```diff
@@ -60,10 +64,13 @@ main {
         width: var(--asset-thumbnail-size);
         height: var(--asset-thumbnail-size);
         object-fit: contain;
-        border-width: 3px !important;
+        border-color: #808080 !important;
```

Comment:
While we're working on this, do you think these should be variables because we're going to reuse them elsewhere and/or do color math on them?

Reply:
I think when we import all Bootstrap scss files, we could use the gray color themes like `$gray-400`.  But yeah I'll create the variable specifically for the prototype for now so it can replaced easily later when we install full Bootstrap.

Reply:
We have all of the bootstrap variables imported now. If there's anything else we need we can add it now as well.